hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/RatisOverReplicationHandler.java

```diff
@@ -321,4 +325,43 @@ private int createCommands(
     return commandsSent;
   }
 
+  static class EligibleAndReserveReplicas {
+    private List<ContainerReplica> eligibleReplicas;
+    private List<ContainerReplica> reserveReplicas;
```

Comment:
I think `reserved` is fine, but might want to consider to use "saved" instead by renaming it to `savedReplicas` or `savedReplicasWithUniqueOrigins` so that it's consistent with the method `saveReplicasWithUniqueOrigins` and reduce the number of special terms in RM.